cartography/intel/gcp/__init__.py

```diff
@@ -513,6 +513,7 @@ def _sync_multiple_projects(
             )
             run_analysis_job(
                 'gcp_storage_bucket_policy_analysis.json',
+                'gcp_compute_network_interface_analysis.json',
```

Comment:
Create separate function call for each job

Reply:
okay